loadtest/control/utils.go

```diff
@@ -57,15 +57,35 @@ func RandomizeUserName(name string) string {
 }
 
 func emulateUserTyping(t string, cb func(term string) UserActionResponse) UserActionResponse {
+	typingSpeed := time.Duration(100+rand.Intn(20)*10) * time.Millisecond // 100-300ms
+
+	typeCorrect := time.NewTicker(typingSpeed)
+	defer typeCorrect.Stop()
+	typeWrong := time.NewTicker(time.Duration(2+rand.Intn(8)) * typingSpeed)
+	defer typeWrong.Stop()
```

Comment:
I am of the opinion that whether or not the user mistypes should be decided randomly at each iteration.
The final result might be similar but I think the logic would be easier to understand. 
So we would end up with one timer only and divide the two cases in a `if rand.Float64() > 0.XX` block.
This is just an idea, willing to discuss it further.

Reply:
Yes, at the end of the day they would be similar. For me selecting between two channels is more expressive than a single if statement but it's me. But I think it can look complicated too.